### DISCUSSION TITLE
【委托】从空洞出来后发现竟然把齿轮硬币带出来了

### DISCUSSION THREAD

**Discussion by ran-mitake-ag** [share121/inter-knot#508] [其他]
如题。我是一个普通的盗洞客，每天干干活就拿着新手绳匠1000丁尼的萝卜回家，但是今天不知道萝卜怎么了，把我带到了一个没人见过的出口，联系绳匠后说是好像在一个帮派里，我也不管进不进局子了，也是直接联系了治安官，打完电话后发现兜里有两枚齿轮硬币，在线等，怎么办，来绳匠，网友都行。

> 解决丁尼不够，拿齿轮硬币付钱`

**Comment by ran-mitake-ag** [comment]
我再也不买便宜萝卜了呜呜

**Comment by warning-Kirov-reporting** [comment]
齿轮硬币似乎只是虚拟货币，推测萝卜出问题了？！

**Comment by ran-mitake-ag** [comment]
齿轮硬币好像有实体，每次正常出来邦布商人会把硬币收走换丁尼，难道我们的习俗不同？
治安官快来了，等我消息

**Comment by ran-mitake-ag** [comment]
呜呜治安官迷路了我出1599菲林995
